app/src/main/res/values/strings.xml

```diff
@@ -1387,8 +1387,9 @@
     <string name="summary_show_home_on_boot">Bring up xDrip screen automatically after device boots</string>
     <string name="title_show_home_on_boot">Show xDrip on Boot</string>
     <string name="eula">EULA</string>
-    <string name="summary_plus_accept_follower_actions">Treatments and Calibrations from followers will be accepted</string>
+    <string name="summary_plus_accept_follower_actions">Treatments, Calibrations and remote sensor stop from followers will be accepted</string>
     <string name="title_plus_accept_follower_actions">Accept follower actions</string>
+    <string name="not_follower_switch">Follower actions switch is disabled</string>
```

Comment:
This message could probably be clearer as well. Also, this is only being used when a follower attempts to stop a sensor, right? Not any of the other locations where a follower performs a disallowed action?

Perhaps "Rejected action from follower because follower actions are disabled", in case you want to future-proof this message to use it in the other contexts as well.

Reply:
Yes, this is a log on the master only when stop sensor is rejected.  Updated.  Thanks